source/agora/consensus/Validation.d

```diff
@@ -988,6 +988,13 @@ public string isInvalidReason (const ref Block block, in ulong prev_height,
     if (block.header.merkle_root != Block.buildMerkleTree(block.txs, merkle_tree))
         return "Block: Merkle root does not match header's";
 
+    foreach (const ref enrollment; block.header.enrollments)
+    {
+        if (auto fail_reason = enrollment.isInvalidEnrollmentReason(
```

Comment:
As you can see, this is never tested

Reply:
I added unittest.
Further verification is likely to be needed.
For example, the sort order of the enrollments.
I will write more unnittest code.

Reply:
I added isStrictlyMonotonic check logic.

Reply:
Oh, I checked the missing unnittest.
A test of this requires #493.
I'll block this for a moment.

Reply:
I think it's settled. Is it correct to check the red line in the cov?